### PR TITLE
Added validation to widget of type 'metrics' to force fromversion 6.2.0

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: https://github.com/demisto/etc/issues/38952

## Description
Add validation to widget of type 'metrics' to force fromversion 6.2.0.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -988,6 +988,7 @@ class PB_Status:
 MARKETPLACE_LIVE_DISCUSSIONS = \
     'https://live.paloaltonetworks.com/t5/cortex-xsoar-discussions/bd-p/Cortex_XSOAR_Discussions'
 MARKETPLACE_MIN_VERSION = '6.0.0'
+WIDGET_TYPE_METRICS_MIN_VERSION = '6.2.0'
```
Comment (roysagi) [inline demisto_sdk/commands/common/constants.py:991]: lets keep this in the widget validator

File: demisto_sdk/commands/common/errors.py [hunk 2]
```diff
@@ -795,6 +796,12 @@ def remove_field_from_widget(field, widget):
     def include_field_in_widget(field, widget_name):
         return f'The field {field} needs to be included in the widget: {widget_name}. Please add it.'
 
+    @staticmethod
+    @error_code_decorator
+    def invalid_fromversion_for_type_metrics(file_path):
+        return f'The minimal fromVersion for widget with data type \'metrics\' is \'6.2.0\'.\n' \
+               f'You can run \'demisto-sdk format -i {file_path}\'.'
```
Comment (roysagi) [inline demisto_sdk/commands/common/errors.py:803]: use `suggeset_fix` and remove this line

File: demisto_sdk/commands/common/tests/widget_test.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        structure.prev_ver = 'master'
+        structure.branch_name = ''
+        structure.quite_bc = quite_bc
+        return structure
+
+
+@pytest.mark.parametrize('current_file, answer', [({'dataType': 'metrics', 'fromVersion': '6.2.0'}, True),
+                                                  ({'dataType': 'metrics', 'fromVersion': '5.5.0'}, False),
+                                                  ({'dataType': 'incidents', 'fromVersion': '6.2.0'}, True)])
+def test_is_valid_fromversion(current_file, answer):
```
Comment (roysagi) [inline demisto_sdk/commands/common/tests/widget_test.py:26]: add docstring

**Review by roysagi — APPROVED** [review]
Looks good please see my comments and add a changelog entry

File: demisto_sdk/commands/common/errors.py (hunk 2 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/common/errors.py:803]: done.

File: demisto_sdk/commands/common/tests/widget_test.py (hunk 3 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/common/tests/widget_test.py:26]: done.

File: demisto_sdk/commands/common/constants.py (hunk 1 above)
Comment (yaakovpraisler) [reply demisto_sdk/commands/common/constants.py:991]: done.

**Pull request merged**